terraform/main.tf

```diff
@@ -1,3 +1,13 @@
+#Including S3 backend
+terraform {
+  backend "s3" {
+    bucket = "xxx"
+    key    = "terraform.tfstate"
+    region = "ap-southeast-2"
```

Comment:
if we are using dynamo db i think we need to specify here. We can do that later
https://medium.com/@mitesh_shamra/state-management-with-terraform-9f13497e54cf#:~:text=Locking%20helps%20make%20sure%20that,S3%20to%20store%20state%20files.